src/components/vm/vmMigrateDialog.jsx

```diff
@@ -102,7 +102,7 @@ const StorageRow = ({ storage, setStorage }) => {
                                         <p>{_("Full disk images and the domain's memory will be migrated. Only non-shared, writable disk images will be transferred. Unused storage will remain on the origin after migration.")}</p>
                                     </FlexItem>
                                 </Flex>}>
-                           <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                           <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230704-092041-0b751cdc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)